packages/support/src/Concerns/Macroable.php

```diff
@@ -39,6 +39,11 @@ public static function flushMacros(): void
         static::$macros = [];
     }
 
+    public static function hasMacro(string $name): bool
+    {
+        return (bool) static::getMacro($name);
```

Comment:
Doing it this way (instead of with some `isset()` calls) ensure that we don't need to repeat the logic for checking the macros on parent classes, which I don't think is great to copy and maintain twice.